draft-ietf-dnsop-svcb-https.md

```diff
@@ -1021,8 +1042,9 @@ the client SHOULD construct a corresponding "https" URL as follows:
 
 This construction is equivalent to Section 8.3 of {{HSTS}}, point 5.
 
-If an HTTPS RR query for this "https" URL returns any HTTPS RRs
-(AliasForm or ServiceForm), the client SHOULD act as if it has received an
+If an HTTPS RR query for this "https" URL returns any AliasForm HTTPS RRs,
+or any compatible ServiceForm HTTPS RRs (see {{mandatory}}), the client
```

Comment:
Should explicitly state that if the set of compatible ServiceForm HTTPS RRs is empty (such as due to all requiring a mandatory key not supported) that clients MUST NOT apply the HSTS-like behavior?  (It's implicit here, but may be worth calling out outright.)

Reply:
Done